### PR TITLE
Add community projects page

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

**Comment by kassane** [comment]
Nice! This matches one of the FAQ answers on tb-cpp.

- https://github.com/kassane/tigerbeetle-cpp#is-this-project-an-official-binding-for-tigerbeetledb

File: docs/COMMUNITY_PROJECTS.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+This page celebrates the incredible work done by community members. 😊
+
+If you've made a project and would like to be featured here, open a
+pull request for this page!
+
+## Applications in the Wild
+
+* [Plain Text Accounting with TigerBeetle](https://github.com/pondersource/pta-tb-experiment#pta-tb-experiment)
+
+## Database Clients Made With ❤️  By the Community
```
Comment (jorangreef) [inline docs/COMMUNITY_PROJECTS.md]:
```suggestion
## Database Clients Made With ❤️ By the Community
```

Remove double space.

File: docs/COMMUNITY_PROJECTS.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Community Projects
+
+This page celebrates the incredible work done by community members. 😊
```
Comment (jorangreef) [inline docs/COMMUNITY_PROJECTS.md]:
```suggestion
Documenting the incredible work done by community members. 😊
```

To also remove one of the "this page" phrases, which occurs again in the second paragraph.

File: docs/COMMUNITY_PROJECTS.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pull request for this page!
+
+## Applications in the Wild
+
+* [Plain Text Accounting with TigerBeetle](https://github.com/pondersource/pta-tb-experiment#pta-tb-experiment)
+
+## Database Clients Made With ❤️  By the Community
+
+They are not officially supported by TigerBeetle but we do our best to
+support them anyway!
```
Comment (jorangreef) [inline docs/COMMUNITY_PROJECTS.md]: Third-party clients not yet officially supported by TigerBeetle.

File: docs/COMMUNITY_PROJECTS.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Keep in mind before our first release there will be frequent breaking
```
Comment (jorangreef) [inline docs/COMMUNITY_PROJECTS.md]:
```suggestion
Keep in mind that before our first release there will be frequent breaking
```

File: docs/COMMUNITY_PROJECTS.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+changes. We integration-test all first-party clients but third-party
+clients may lag behind.
+
+* [tigerbeetlex](https://github.com/rbino/tigerbeetlex): An Elixir client for TigerBeetle
+* [tigerbeetle-cpp](https://github.com/kassane/tigerbeetle-cpp): TigerBeetle C++ Client
+* [tigerbeetle-rs](https://github.com/ZetaNumbers/tigerbeetle-rs): Safe bindings to TigerBeetle
```
Comment (jorangreef) [inline docs/COMMUNITY_PROJECTS.md]:
```suggestion
* [tigerbeetlex](https://github.com/rbino/tigerbeetlex): An Elixir client
* [tigerbeetle-cpp](https://github.com/kassane/tigerbeetle-cpp): A C++ client
* [tigerbeetle-rs](https://github.com/ZetaNumbers/tigerbeetle-rs): A Rust client
```

File: docs/COMMUNITY_PROJECTS.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+Need a client for Go, Java, Node.js, or .NET? We've got you covered
+with [first-party
+clients](https://github.com/tigerbeetledb/tigerbeetle/tree/main/src/clients).
+
+## Other
+
+* [A Jepsen Test for TigerBeetle](https://github.com/nurturenature/jepsen-tigerbeetle)
```
Comment (jorangreef) [inline docs/COMMUNITY_PROJECTS.md]: This could (with 50/50 probability) be moved under Applications in the Wild.

**Review by jorangreef — DISMISSED** [review]

File: docs/COMMUNITY_PROJECTS.md (hunk 6 above)
Comment (eatonphil) [reply docs/COMMUNITY_PROJECTS.md]: Done!

File: docs/COMMUNITY_PROJECTS.md (hunk 3 above)
Comment (eatonphil) [reply docs/COMMUNITY_PROJECTS.md]: Done!

**Comment by eatonphil** [comment]
Hey @batiati, this was already approved just looking for a reapproval. Thanks!

**Review by batiati — APPROVED** [review]
That is a nice list, can't wait to see it growing and growing 🌱

**Comment by eatonphil** [comment]
Thanks Rafael!

**Pull request merged**